Storage/StorageTests/CoreData/CoreDataIterativeMigratorTests.swift

```diff
@@ -7,10 +7,10 @@ import CoreData
 final class CoreDataIterativeMigratorTests: XCTestCase {
     private var modelsInventory: ManagedObjectModelsInventory!
 
-    override func setUp() {
-        super.setUp()
+    override func setUpWithError() throws {
```

Comment:
I really am happy with this new pattern allowing setUp and tearDown to have failures and they're presented properly. 😄